client/public/index.html

```diff
@@ -21,7 +21,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-  <title>Uni Solutiuns</title>
+  <title>Uni Solutions</title>
```

Comment:
Tego typu rzeczy raczej warto jest robić w oddzielnych PRach. Bo jak np coś we slajderze będzie nie halo i trzeba będzie go zrevertować to poprawka tej pierdoły nie zostanie cofnięta, a wiadomo, że chcemy by w źródłach to zostało